FreeRTOS/Demo/Common/include/semtest.h

```diff
@@ -29,7 +29,7 @@
 #define SEMAPHORE_TEST_H
 
 void vStartSemaphoreTasks( UBaseType_t uxPriority );
-BaseType_t xAreSemaphoreTasksStillRunning( void );

```

Comment:
Thank you for fixing this inconsistency. 

I may want to update your PR to use ```BaseType_t``` in both ```semtest.{c,h}```. Since according to https://www.freertos.org/upgrading-to-FreeRTOS-V8.html, ```portBASE_TYPE``` is the old name, and ```BaseType_t``` is the new name (since FreeRTOS V8.x.x ).